include/rfb-proto.h

```diff
@@ -144,7 +144,14 @@ struct rfb_client_cut_text_msg {
 	uint8_t type;
 	uint8_t padding[3];
 	uint32_t length;
-	char test[0];
+	char data[0];
+} RFB_PACKED;
+
+struct rfb_server_cut_text_msg {
```

Comment:
These structs are the same, right? Why not just have one of them called `rfb_cut_text_msg`?